awx/ui_next/src/components/Search/Search.jsx

```diff
@@ -175,10 +185,16 @@ class Search extends React.Component {
 
         if (Array.isArray(queryParams[key])) {
           queryParams[key].forEach(val =>
-            queryParamsByKey[columnKey].chips.push(val.toString())
+            queryParamsByKey[columnKey].chips.push({
+              key: `${key}:${val}`,
+              node: <span>{getLabelFromValue(val, columnKey)}</span>,
```

Comment:
I think you can omit the `<span>` wrapper here and still get the same result (unless that produces prop type errors?)